### ISSUE TITLE
License Choice

### ISSUE THREAD

**Issue by sreejithmunthikodu** [UBC-MDS/KmeansR#54]
Guys, lets discuss the choice of our license here.
**Here are some of the popular OSI-approved licenses** [Source](https://opensource.org/licenses)   
- Apache License 2.0
- BSD 3-Clause "New" or "Revised" license
- BSD 2-Clause "Simplified" or "FreeBSD" license
- GNU General Public License (GPL)
- GNU Library or "Lesser" General Public License (LGPL)
- MIT license
- Mozilla Public License 2.0
- Common Development and Distribution License
- Eclipse Public License version 2.0

**Comment by sreejithmunthikodu** [comment]
I believe MIT license is best suited for us as it is very simple, to the point and lets the user do almost anything with our project.

**Comment by RobBlumberg** [comment]
I agree, I'll also just elaborate on that slightly by saying that we, in fact, actively encourage modification and reproduction of our code since this package is meant for learning purposes, and so the MIT license allows for that flexibility, without holding us liable to any subsequent modifications.

**Comment by saurav193** [comment]
Also we can add, since our package is not novel work but just a reproduction of a pre-existing package, we are not adding some extra functionality to python. This package is useful for learning purposes only and well suited for beginners who are starting to build their package for python. That is why we are allowing others to distribute, modify or improve our code without our permission.
